### DISCUSSION TITLE
Image 404 in the console log : ?_rsc=404 (Not Found)

### DISCUSSION THREAD

**Discussion by MonaAghili** [vercel/next.js#62990] [Help]
### Summary

Despite the image appearing on the site, it reports an error code of 404 in the console ?_rsc=404 (Not Found)
There is ?_rsc at the end of the Image URL path

**Comment by ariannargesi** [comment]
Can you please reproduce the error in a Codesandbox? @MonaAghili

**Comment by shriekdj** [comment]
are you using custom server for running nextjs?
